x/wasm/types/types.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+package wasmd.x.wasmd.v1;
+
+option go_package = "github.com/cosmwasm/wasmd/x/wasmd/types";
+
+import "third_party/proto/gogoproto/gogo.proto";
+import "types/types.proto";
+
+message MsgStoreCode{
+    option (gogoproto.goproto_getters) = false;
+
+    bytes sender = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    // WASMByteCode can be raw or gzip compressed
+    bytes wasm_byte_code = 2 [(gogoproto.moretags) = "yaml:\"wasm_byte_code\""];
+    // Source is a valid absolute HTTPS URI to the contract's source code, optional
+    string source = 3;
+    // Builder is a valid docker image name with tag, optional
+    string builder = 4;
+}
+
+message MsgInstantiateContract{
+    option (gogoproto.goproto_getters) = false;
+
+    bytes sender = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    uint64 code = 2 [(gogoproto.moretags) = "yaml:\"code_id\""];
+    string label = 3;
+    bytes init_msg = 4 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+    repeated cosmos_sdk.v1.Coin init_funds = 5 [
```

Comment:
Where is this imported from by the way?

Is there a good link on how to compile this (which tooling)? (So I can modify the proto if needed)

Reply:
you need to run make proto-gen, so that it generates types.pb.go